b2sdk/api.py

```diff
@@ -127,37 +127,37 @@ def authorize_automatically(self):
         try:
             self.authorize_account(
                 self.account_info.get_realm(),
-                self.account_info.get_account_id_or_app_key_id(),
+                self.account_info.get_application_key_id(),
                 self.account_info.get_application_key(),
             )
         except MissingAccountData:
             return False
         return True
 
     @limit_trace_arguments(only=('self', 'realm'))
-    def authorize_account(self, realm, account_id_or_key_id, application_key):
+    def authorize_account(self, realm, application_key_id, application_key):
         """
         Perform account authorization
 
         :param realm: a realm to authorize account in (usually just "production")
         :type realm: str
-        :param account_id_or_key_id: account or key ID
-        :type account_id_or_key_id: str
+        :param application_key_id: application key ID
+        :type application_key_id: str
         :param application_key: user's application key
         :type application_key: str
         """
         # Clean up any previous account info if it was for a different account.
         try:
-            old_account_id = self.account_info.get_account_id()
+            old_id = self.account_info.get_application_key_id()
```

Comment:
Woops this was supposed to be in a different PR

Reply:
Let me know if you feel strongly about this in a different PR and I can remove it and maybe add a test to make sure the cache is cleared?